setup.py

```diff
@@ -14,7 +14,7 @@
 def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
-install_requires = ['aiohttp>=3.0.1']
+install_requires = ['aiohttp>=3.0.1', 'typing_extensions>=3.6.5']
```

Comment:
Only in Python <3.8.
See: https://github.com/aio-libs/aiohttp/blob/3.8/setup.py#L73